.env

```diff
@@ -33,6 +33,9 @@ LOKI_CONFIG_PATH=/etc/loki
 LOKI_HOST=loki
 LOKI_PORT=3100
 
+TEMPO_CONFIG_FILE=tempo.yaml
+TEMPO_CONFIG_PATH=/etc/tempo-config
+TEMPO_CONFIG_PATH_LOCAL=./infra/tempo/config
 TEMPO_HOST=tempo
 TEMPO_PORT=3200
 TEMPO_PORT_OTLP_RECEIVER=4317
```

Comment:
The following lines can also be deleted as there's no real need to have them here as long as they are in the hardcoded in the config. It kinda made sense in `docker-compose` tho'

```
TEMPO_TEMP_PATH=/tmp/tempo
TEMPO_TRACES_PATH=traces
TEMPO_WAL_PATH=wal
```